### ISSUE TITLE
Anime

### ISSUE THREAD

**Issue by MrLucado** [Floya-dev/VencordThemeForAnna#1]
This would be for the Kaichou Wa Maid-Sama! Manga, lol.

**Comment by Floya-dev** [comment]
Yeah Iam not really sure which anime it is, so I didn't want to put the name in the description.
I don't watch anime.
